libpam/include/security/_pam_macros.h

```diff
@@ -94,11 +94,31 @@ do {                                              \
 #define _PAM_LOGFILE "/var/run/pam-debug.log"
 #endif
 
+#ifdef PAM_NO_HEADER_FUNCTIONS
+#ifdef PAM_DEBUG_C
+#define PAM_DEBUG_TYPE void
+#else
+#define PAM_DEBUG_TYPE extern void
+#endif
+UNUSED
+PAM_DEBUG_TYPE _pam_output_debug_info(const char *file, const char *fn
+				      , const int line);
+UNUSED
+PAM_FORMAT((printf, 1, 2))
+PAM_DEBUG_TYPE _pam_output_debug(const char *format, ...);
+#undef PAM_DEBUG_TYPE
+#else
+#ifdef PAM_DEBUG_C
+#define PAM_DEBUG_TYPE void
+#else
+#define PAM_DEBUG_TYPE static void
+#endif
```

Comment:
I suggest not to include `void` into PAM_DEBUG_TYPE.
Let's rename it to `PAM_DEBUG_SCOPE` or something like this because it's all about presence or absence of `static` keyword.

Reply:
Yes, SCOPE sounds better here. Adjusted.

Reply:
I still suggest not to include `void` into `PAM_DEBUG_SCOPE`, let it be just `static` or nothing.

Reply:
And adjusted as well